app/code/local/Magmodules/Webwinkelconnect/Model/Api.php

```diff
@@ -159,4 +154,33 @@ public function getStoreIds()
         return $storeIds;
     }
 
+    private function getOrderProducts(Mage_Sales_Model_Order $order): array {
+        $products = [];
+        foreach ($order->getAllItems() as $item) {
+            $product = $item->getProduct();
+            $products[] = [
+                'name' => $product->getName(),
+                'url' => $product->getProductUrl(),
+                'id' => $product->getEntityId(),
+                'sku' => $product->getSku(),
+                'image_url' => '',
+                'brand' => $product->getAttributeText('manufacturer'),
+            ];
+        }
+
+        return $products;
+    }
+
+    private function getLanguage(int $storeId, Mage_Sales_Model_Order $order): string {
+        $language = Mage::getStoreConfig('webwinkelconnect/invitation/language', $storeId);
+        if (empty($language) || $language == '') {
```

Comment:
```suggestion
        if (empty($language)) {
```

Empty already covers `''`